### DISCUSSION TITLE
Search nested doesn't deal with ambiguities

### DISCUSSION THREAD

**Discussion by RulianZB** [tailflow/laravel-orion#280] [Help]
Hi,
When i try to use an orion search request with nested filters like this,

`"filters": [
        {"field": "site.id", "operator": "=", "value": IdSite},
        {"type": "or", "nested" : [
            {"field" : "updated_at", "operator": ">=", "value": "05-06-2024"},
            {"type":"or", "field" : "pages.modify_at", "operator": ">=", "value": "05-06-2024"},
            {"type":"or", "field" : "pages.questions.modify_at", "operator": ">=", "value": "05-06-2024"},
            {"type":"or", "field" : "pages.questions.questionChoices.modify_at", "operator": ">=", "value": "05-06-2024"},
            {"type":"or", "field" : "pages.questions.conditions.modify_at", "operator": ">=", "value": "05-06-2024"},
            {"type":"or", "field" : "pages.questions.questionChoices.conditions.modify_at", "operator": ">=", "value": "05-06-2024"}
        ]}
    ]`

I got a problem of integrity constraint, which means : "Integrity constraint violation: 1052 Column 'modify_at' in where clause is ambiguous"
Someone has an idea to how to solve it ?

**Comment by alexzarbn** [comment]
Hi @RulianZB,

Thank you for reporting this. I will take a look.

**Comment by alexzarbn** [comment]
@RulianZB Could you also specify what type of relation the `pages` -> `questions`, `questions` -> `questionChoices`, and `questionChoices` -> `conditions` are?

**Comment by RulianZB** [comment]
Sorry for not responding sooner,

pages -> questions => morphMany(Question::class, 'questionable');

questions -> questionChoices => hasMany(QuestionChoice::class)->orderBy('order');

questionChoices -> conditions => belongsToMany(Question::class, 'condition')->using(Condition::class)
